packages/forms-web-app/src/lib/decode-string.js

```diff
@@ -1,7 +1,13 @@
 const decodeString = (stringToDecode) => {
 	if (!stringToDecode || typeof stringToDecode !== 'string') return '';
 
-	const decodedString = decodeURIComponent(stringToDecode);
+	let decodedString = '';
+
+	try {
+		decodedString = decodeURIComponent(stringToDecode);
+	} catch {
+		decodedString = stringToDecode;
```

Comment:
Would be the optimal expected behaviour to return the primary payload if it couldn't be decoded (catch error)?

Reply:
This fixes an error when an invalid input was being decoded (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Malformed_URI). In user cases we still want the string but for the decode/encode to not take place. Looking into creating checks for valid strings looks quite cumbersome. This was the most elegant solution that I could find.